jcl/src/java.base/share/classes/java/lang/invoke/InfoFromMemberName.java

```diff
@@ -21,3 +21,47 @@
  *
  * SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
  *******************************************************************************/
+/*[IF Java15]*/
+package java.lang.invoke;
+
+import java.lang.invoke.MethodHandles.Lookup;
+import java.lang.reflect.Member;
+
+class InfoFromMemberName implements MethodHandleInfo {
+	InfoFromMemberName(Lookup lookup, MemberName member, byte kind) {
+		throw OpenJDKCompileStub.OpenJDKCompileStubThrowError();
+	}
+
+	@Override
+	public Class<?> getDeclaringClass() {
+		throw OpenJDKCompileStub.OpenJDKCompileStubThrowError();
+	}
+
+	@Override
+	public String getName() {
+		throw OpenJDKCompileStub.OpenJDKCompileStubThrowError();
+	}
+
+	@Override
+	public MethodType getMethodType() {
+		throw OpenJDKCompileStub.OpenJDKCompileStubThrowError();
+
+	}
+
+	@Override
+	public int getModifiers() {
+		throw OpenJDKCompileStub.OpenJDKCompileStubThrowError();
+	}
+
+	@Override
+	public int getReferenceKind() {
+		throw OpenJDKCompileStub.OpenJDKCompileStubThrowError();
+	}
+
+	@Override
+	public <T extends Member> T reflectAs(Class<T> expected, Lookup lookup) {
+		throw OpenJDKCompileStub.OpenJDKCompileStubThrowError();
+	}
+}
+/*[ENDIF] Java15 */
+
```

Comment:
Now it seems there is an extra blank line.

Reply:
I thought the previous feedback https://github.com/eclipse/openj9/pull/9865#discussion_r442983388 had requested `a blank line`. I am hoping this is fine.

Reply:
There doesn't need to be a blank line, we just like to have a linefeed at the end of the last line, and there wasn't one originally.